entities/schema/accessors.go

```diff
@@ -24,6 +24,10 @@ func (s *Schema) GetClass(className ClassName) *models.Class {
 	return class
 }
 
+func (s *Schema) ReadOnlyClass(name string) *models.Class {
```

Comment:
maybe I am missing something, but how is this different than using GetClass directly? Or is it just a helper function to signal "do not write to this class"?

Reply:
it's just helper to  be used for read situations and without principal checks

Reply:
but it does exactly the same as GetClass directly above? So what is the point of adding it?

Reply:
to satisfy the finder [interface](https://github.com/weaviate/weaviate/pull/4016/files#diff-f0c4ee4d5df4c750eba95199d59c85e6c4f81fd1a55be424c237580aae9d094dR21) and `GetClass`is used somewhere else with different params, i agree it's a bit confusing. gonna try to find another way